package.json

```diff
@@ -9,7 +9,7 @@
     "@types/jest": "^26.0.24",
     "@types/react": "^17.0.14",
     "@types/segment-analytics": "^0.0.34",
-    "jest": "^27.0.6",
+    "jest": "^27.3.1",
```

Comment:
Update to support node protocol imports from some deps:
```
import fs from 'node:fs'
```

Reply:
💭 Never seen `from 'node:fs'` before!